### ISSUE TITLE
Don't use spread operator for Objects

### ISSUE THREAD

**Issue by ibc** [versatica/pick-port#1]
```js
const _options = { ...defaultOptions, ...options };
```

```
/xxxx/node_modules/pick-port/index.js:51
	const _options = { ...defaultOptions, ...options };
	                   ^^^

SyntaxError: Unexpected token ...
```

Spread operators it's not defined inES6 and it fails in Node 6 and 8:

* https://stackoverflow.com/questions/40066731/node-v6-failing-on-object-spread
* http://node.green/

**Comment by ibc** [comment]
Instead, use `Object.assign`. Also, let's avoid creating a ugly `_options` variable:

```js
options = Object.assign({}, defaultOptions, options);
```

**Comment by jmillan** [comment]
Addressed in 0.0.3. Thanks @ibc!
